sphinxgallery/gen_rst.py

```diff
@@ -225,7 +225,7 @@ def _plots_are_current(src_file, image_file):
     return not needs_replot
 
 
-def save_figures(image_path, fig_count):
+def save_figures(image_path, fig_count, gallery_conf):
```

Comment:
ping @Titan-C, do you know off the top of your head if there is a way to access the sphinx application config without having to pass it around as an argument in each function like this?